packages/core/src/eth/coin.ts

```diff
@@ -18,7 +18,9 @@ export abstract class Coin {
 
   protected constructor(coinData: CoinData) {
     this.coinData = coinData;
-    this.walletContract = new new Web3().eth.Contract(walletAbi as AbiItem[]);
+    this.walletContract = new new Web3().eth.Contract(
```

Comment:
could you explain why you typecast walletContract to "any"?

Reply:
fixed it @cloudinertia